DUI3-DX/Connectors/Revit/Speckle.Connectors.Revit2023/Speckle.Connectors.Revit2023.csproj

```diff
@@ -34,6 +32,16 @@
     <PackageReference Include="Speckle.Revit.API" Version="2023.0.0" IncludeAssets="compile;build" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Include="..\Speckle.Connectors.RevitShared\**\*.cs"  Exclude="..\Speckle.Connectors.RevitShared\obj\**\*.*"/>
+  </ItemGroup>
```

Comment:
I've never had to run this Exclude attribute for the `obj` folder. 🤔

It kinda feels wrong

Reply:
making a csproj just for shared makes an obj folder.

we don't strictly need the csproj for shared (since each individual "real" csproj includes the files separately) but it's making the shared stuff viewable in the sln